packages/backend-deployer/src/cdk_deployer.test.ts

```diff
@@ -124,8 +122,6 @@ void describe('invokeCDKCommand', () => {
     assert.strictEqual(deployMock.mock.callCount(), 1);
     assert.deepStrictEqual(deployMock.mock.calls[0].arguments[1], {
       hotswap: HotswapMode.FALL_BACK,
-      ci: false,
-      requireApproval: undefined,
```

Comment:
These were already deactivated/not used in the current versions. 0.2.0 explicitly removes them. These functionalities are built in the `IoHost`